data/json/items/vehicle/boat.json

```diff
@@ -31,6 +47,22 @@
     "to_hit": -1,
     "category": "veh_parts"
   },
+  {
+    "type": "GENERIC",
+    "id": "fold_plastic_boat_hull",
+    "symbol": "o",
+    "color": "brown",
+    "name": { "str": "folding plastic boat hull" },
+    "description": "A rigid plastic sheet that keeps the boat afloat. It can be folded. Add boat hulls to a vehicle until it floats.  Then attach oars or a motor to get the boat to move.",
```

Comment:
```suggestion
    "description": "A rigid plastic sheet that keeps the boat afloat.  It can be folded.  Add boat hulls to a vehicle until it floats.  Then attach oars or a motor to get the boat to move.",
```